nnvm/tests/python/frontend/tensorflow/test_forward.py

```diff
@@ -198,6 +210,66 @@ def test_forward_pooling():
                  dilation_rate=[1, 1],
                  strides=[1, 2])
 
+    if is_gpu_available():
+        _test_pooling(input_shape=[2, 2, 9, 10],
+                     window_shape=[1, 1],
+                     padding='SAME',
+                     pooling_type='MAX',
+                     dilation_rate=[1, 1],
+                     data_format="NCHW",
+                     strides=[1, 1])
+        _test_pooling(input_shape=[2, 2, 9, 10],
+                     window_shape=[1, 1],
+                     padding='SAME',
+                     pooling_type='AVG',
+                     dilation_rate=[1, 1],
+                     data_format="NCHW",
+                     strides=[1, 1])
+
+        _test_pooling(input_shape=[2, 2, 10, 9],
+                     window_shape=[1, 1],
+                     padding='SAME',
+                     pooling_type='MAX',
+                     dilation_rate=[1, 1],
+                     data_format="NCHW",
+                     strides=[1, 1])
+        _test_pooling(input_shape=[2, 2, 10, 9],
+                     window_shape=[1, 1],
+                     padding='SAME',
+                     pooling_type='AVG',
+                     dilation_rate=[1, 1],
+                     data_format="NCHW",
+                     strides=[1, 1])
+
+        _test_pooling(input_shape=[2, 2, 9, 10],
+                     window_shape=[2, 1],
+                     padding='SAME',
+                     pooling_type='MAX',
+                     dilation_rate=[1, 1],
+                     data_format="NCHW",
+                     strides=[1, 1])
+        _test_pooling(input_shape=[2, 2, 9, 10],
+                     window_shape=[2, 1],
+                     padding='SAME',
+                     pooling_type='AVG',
+                     dilation_rate=[1, 1],
+                     data_format="NCHW",
+                     strides=[2, 1])
+
+        _test_pooling(input_shape=[2, 2, 10, 9],
+                     window_shape=[2, 3],
+                     padding='SAME',
+                     pooling_type='MAX',
+                     dilation_rate=[1, 1],
+                     data_format="NCHW",
+                     strides=[2, 1])
+        _test_pooling(input_shape=[2, 2, 10, 9],
+                     window_shape=[2, 3],
+                     padding='SAME',
+                     pooling_type='AVG',
+                     dilation_rate=[1, 1],
+                     data_format="NCHW",
```

Comment:
suggest to use loops and make the _test_pooling testcases compact. 
```pooling = ['AVG', 'MAX']
for pool in pooling:
  _test_pooling(,pool, , , )```
similarly can be done for other parameters also.

Reply:
Done